src/game.rs

```diff
@@ -65,14 +65,12 @@ pub mod resources {
 
     /// Retrieve the string keys of this object.
     pub fn keys() -> Vec<IntershardResourceType> {
-        js_unwrap!(Object
-            .keys(Game.resources)
-            .map(__intershard_resource_type_str_to_num))
+        js_unwrap!(Object.keys(Game.resources).map(__resource_type_str_to_num))
```

Comment:
I'm surprised `rustfmt` changed its mind on that one.

Reply:
I think this might be because I'm not on the nightly?

Reply:
Ahh, shorter js function name pushed it back under the one line char limit.